src/pages/iou/request/step/IOURequestStepScan/index.native.tsx

```diff
@@ -245,32 +246,95 @@ function IOURequestStepScan({
                     });
                     return;
                 }
-                if (iouType === CONST.IOU.TYPE.TRACK && report) {
-                    IOU.trackExpense(
-                        report,
-                        0,
-                        transaction?.currency ?? 'USD',
-                        transaction?.created ?? '',
-                        '',
-                        currentUserPersonalDetails.login,
-                        currentUserPersonalDetails.accountID,
-                        participants[0],
-                        '',
-                        receipt,
-                    );
-                    return;
-                }
-                IOU.requestMoney(
-                    report,
-                    0,
-                    transaction?.currency ?? 'USD',
-                    transaction?.created ?? '',
-                    '',
-                    currentUserPersonalDetails.login,
-                    currentUserPersonalDetails.accountID,
-                    participants[0],
-                    '',
-                    receipt,
+                getCurrentPosition(
```

Comment:
I haven't worked with `.native` files before so forgive me if this is an ignorant question, but the changes here and in `src/pages/iou/request/step/IOURequestStepScan/index.tsx` are exactly the same character for character. Can this not be extracted out and reused so that we have DRY-er code?

Reply:
We kinda have to split the files when we want different behavior in native and web (in this case, the camera for phones and whatnot). That causes a bunch of duplicate code (you can take a look at the files and you'll see they are very similar).
I think there's an initiative to try and improve this, but, right now, it is what we have 🤷

Reply:
Fair enough! Thanks for the context